metaconfig-tests/shared/src/test/scala/metaconfig/DeriveConfDecoderExSuite.scala

```diff
@@ -30,7 +30,10 @@ class DeriveConfDecoderExSuite extends munit.FunSuite {
   ): Unit =
     decoder.read(Option(in), cfg) match {
       case Configured.NotOk(err) => fail(err.toString)
-      case Configured.Ok(obtained) => assertEquals(obtained, out)
+      case Configured.Ok(obtained) =>
+        // TODO: assertEquals here fails Scala 3 compilation.
```

Comment:
It's fine to remove the TODO comment. I've been meaning to relax the signature on `assertEquals` but i keep delaying it because I don't want to break bincompat in MUnit

Reply:
cool!